.gitpod.yml

```diff
@@ -57,11 +57,12 @@ tasks:
     before: scripts/branch-namespace.sh
     init: yarn --network-timeout 100000 && yarn build
   - name: Go
-    before: pre-commit install --install-hooks
     init: |
       ./components/gitpod-protocol/go/scripts/generate-config.sh
-      leeway exec --filter-type go -v -- go mod verify
+      leeway exec --filter-type go -v -- go mod download
     openMode: split-right
+  - name: Pre-commit
+    init: pre-commit install --install-hooks
```

Comment:
@easyCZ, I wonder, could we install them *and* warm them um in dev image? They don't receive frequent update, do they?

Reply:
Good idea. I'd need to investigate if the image itself has access to the `.pre-commit` config, or if we'd need to shift that into the image also.

Reply:
happy to proceed with this PR as well. just to let you know, if you think this is ready, let us know.